common/meta/types/src/table.rs

```diff
@@ -78,6 +78,9 @@ pub struct TableInfo {
     /// For a table function, it is `table_name(args)`
     pub desc: String,
 
+    /// Database name of this table.
+    pub db: String,
```

Comment:
Why do we need this field?

A user may try to re-fetch a table by `db_name.table_name`, which potentially is an inconsistent operation.

Reply:
Thank you, xp!
If we move the `build_table` to catalog trait, this `db` is not used anymore, I will have a try.